client/jetpack-cloud/sections/auth/controller.tsx

```diff
@@ -31,13 +39,17 @@ export const connect: PageJS.Callback = ( context, next ) => {
 			? `${ protocol }//${ host }:${ port }${ authTokenRedirectPath() }`
 			: `${ protocol }//${ host }${ authTokenRedirectPath() }`;
 
-		const params = {
+		const params: OAuth2Params = {
 			response_type: 'token',
 			client_id: config( 'oauth_client_id' ),
 			redirect_uri: redirectUri,
 			scope: 'global',
 		};
 
+		if ( config.isEnabled( 'oauth_implicit_authorization' ) ) {
```

Comment:
Without this, the backend won't implicitly approve the authorization step. That mean that through this feature flag, we can control in which environments we want this to happen.